### ISSUE TITLE
Расчет повременной зарплаты и оклад

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2118]
Originally reported on Google Code with ID 2118

```
Расчет повременной зарплаты и оклад.  С возможностью добавлять  пользователю различные
надбавки и формулы для их расчета.
```

Reported by `roberto1813` on 2010-09-21 16:19:01

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-09-21 16:26:35
- **Labels added**: Wage, FC2
- **Labels removed**: WAGEFC

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2011-07-25 18:38:40
- **Status changed**: `Duplicate`
- **Merged into**: #1886
